DreamJournal/DreamJournal/HomeView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  HomeView.swift
+//  DreamJournal
+//
+//  Created by CUBS Customer on 11/4/22.
+//
+
+import SwiftUI
+import CoreData
+
+struct HomeView: View {
+    @Environment(\.managedObjectContext) private var viewContext
+    
+    @FetchRequest(entity: Entry.entity(), sortDescriptors: [NSSortDescriptor(key: "date", ascending: false)]) private var allEntries: FetchedResults<Entry>
+    
+    var body: some View {
+        NavigationView {
+            ZStack {
+                LinearGradient(gradient: Gradient(colors: [CustomColor.Navy, CustomColor.SkyPurple]), startPoint: .top, endPoint: .bottom)
+                    .ignoresSafeArea()
+                Image("MoonLogo")
+                    .resizable()
+                    .scaledToFit()
+                    .frame(width:300, height: 300)
+                VStack {
+                    Divider()
+                    Spacer()
+                    List {
+                        ForEach(allEntries) {
+                            entry in HStack {
+                                Circle()
+                                    .frame(width: 15, height: 15)
+                                Spacer().frame(width: 20)
```

Comment:
I could be wrong here but I don't think it's very typical to define the width of a spacer, because spacers are usually used to take up a dynamic amount of space. I think instead of that you could put `.padding(.leading, 20)` on the text element, or trailing space on the circle, depending on your specific intention.

Reply:
That might be better. I'm still working on changing my lists (and truthfully, I've been trying to see if I can make them longer entries/lists to display all of the dream "summaries" rather than just the titles). I'll definitely be changing this!